manticore/core/workspace.py

```diff
@@ -58,6 +58,28 @@ class Store(object):
 
     Implement either save_value/load_value in subclasses, or save_stream/load_stream, or both.
     """
+    @classmethod
+    def store_type(cls):
```

Comment:
I had class variables initially, but wanted to be able to raise when unimplemented. I've reverted to variables and just check in `fromdescriptor` now.